packages/nuxt/src/app/entry.ts

```diff
@@ -3,7 +3,7 @@ import type { App } from 'vue'
 
 // These files must be imported first as they have side effects:
 // 1. (we set __webpack_public_path via this import, if using webpack builder)
-import '#build/paths.mjs'
+import '#paths'
```

Comment:
i'm sure nitro at least will keep using `#internal/nitro/` scoped virtuas but it _might_ conflict with any other alias by modules we could use `#nuxt-paths` or something non conflicting. totally up to you.